### DISCUSSION TITLE
Support pydantic v2

### DISCUSSION THREAD

**Discussion by gbasei** [roman-right/bunnet#13] [Feature Request]
Hi, thank you for this project, we just started using it and it looks very promising.
Do you plan supporting pydantic v2 in Bunnet? I noticed that it is already supported in Beanie.

**Comment by roman-right** [comment]
Hi! Thank you.
Yes, I plan to start the integration of new features to bunnet as soon as I deploy the release version of beanie this week. It will take ~1-2 weeks to merge everything to bunnet.
